.maestro/autofill/3_autofill_prompted_to_save_creds_on_form.yaml

```diff
@@ -1,9 +1,9 @@
 appId: com.duckduckgo.mobile.android
 name: "Autofill: Prompted to save and update credentials on web form"
 tags:
-    - autofillNoAuthTests
+    - autofillNoAuthTestsModernWebView
```

Comment:
Changing this to prevent it from running during autofill nightlies (since Maestro Cloud emulators don't come with a modern `WebView` version, we can't test for actual autofill in the `WebView`)

Reply:
As a follow-up, do you think makes sense to skip our capabilities checks and run it anyways? it should work there

Reply:
I don't think I understand; if the capability checks are telling us it won't run, why would we try it in the UI test?